package.json

```
@@ -20,7 +20,7 @@
     "test": "mocha -r ts-node/register --extensions ts,tsx --timeout 30000 --watch-files src,test 'test/**/*.test.ts'",
     "cli": "ts-node src/bin.ts",
     "build": "rimraf dist dist-esm && tsc && tsc -p tsconfig.esm.json",
-    "prepublishOnly": "tsc"
+    "prepublishOnly": "npm run build"
```

When using `prepack` the result can be easily verified with `npm pack`.

```suggestion
    "prepack": "npm run build"
```

---

Good idea!